File: MusicService.Entities/Entities/Playlist.cs
using MusicWebApp.Core.Abstract;

namespace MusicService.Entities.Entities
{
    public class Playlist : IEntity
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string UserId { get; set; }

        // Navigation properties 
        public virtual ICollection<Music>? Musics { get; set; }
    }
}

File: musicwebapp.uiapp/src/types/Playlist.ts
import { Music } from "./Music";

export type Playlist = {
  id: number; // Corresponds to C# 'int Id'
  name?: string | null; // Corresponds to C# 'string? Name'
  userId: string; // Corresponds to C# 'string UserId'
  musics?: Music[] | null;
};
